content/applications/sales/point_of_sale/payment_methods.rst

```diff
@@ -11,14 +11,8 @@ check :guilabel:`Cash` for cash payments.
 .. image:: payment_methods/payment-method.png
```

Comment:
This image and the instructions are still from 14 - do we want to update it in later versions to match the current process? Something like:

Go to :menuselection:`Point of Sale --> Configuration --> Payment Methods --> New` and set a name. For cash payments, check :guilabel:`Cash`, or to use a **payment terminal**, first select the appropriate :guibel:`Journal`, and then the :guilabel:`Payment Terminal`.

It might also be worth explaining which journal they should select, but I don't know if we want to get into that type of advice, and I'd have to ask the PO to fully understand how to explain it.